composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "codeception/codeception": "2.0.*"
+        "codeception/codeception": "2.0.16"
```

Comment:
I recommend that version number to specify "~2.0.16". it's mean ">=2.0.16, <2.1.0"